src/encoded/static/components/matrix_entex.js

```diff
@@ -164,15 +208,14 @@ const generateColMap = (context) => {
             }
         }
     });
-    return colMap;
-};
 
+    // If targetAssays only contains nulls, then just empty it so we can skip the targetAssay row.
+    if (!targetAssays.some(assay => assay !== null)) {
```

Comment:
It seems like .every is a better option than .some in this case. The inverse of !== gets confusing sometimes.

Reply:
I agree it’s less confusing, but this way the loop will bail early if it finds a non-null assay. With .every(), it always goes through the entire targetAssays array no matter what.